README.md

```diff
@@ -1 +1,25 @@
-# extract-id3-tags-from-mp3s
+<<<<<<< HEAD
+# Extract ID3 Tags From MP3s
+
+>This program extracts id3 tags from an mp3 file, allowing automated metadata collection
+
+## Install
+```sh
+$ npm install extract-id3-tags-from-mp3s
+```
+
+## Usage
+Simply run the program with an mp3 file, such as:
+```sh
+$ node extract-id3-tags-from-mp3s (mp3 file)
+```
+
+## Contribute
+If you'd like to contribute, feel free to do so!
```

Comment:
In general, add a space between the headers and the paragraphs.

Reply:
This is because Markdown will automatically append content after a single new line to the previous paragraph, so it is good practice to use two line breaks each time to make sure that this doesn't happen.